ZlobinSV/lab_1/main.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+int main()
+{
+    int i = 0, zam2, n = 0, k = 0, sum[100];
+    string zam1, str, word[100];
+    char a;
+
+    cout << "Enter the string: ";
+
+    getline(cin, str);
+
+    str += " ";
+
+    while (i < str.length() - 1) {
+        while (str[i] != ' ' && str[i] != ',' && str[i] != '.' && str[i] != '!' && str[i] != '?') {
```

Comment:
Please extract check for punctuation mark into separate function e.g. `bool is_punctuation(char ch);`